presto-kafka/src/main/java/com/facebook/presto/kafka/encoder/raw/RawRowEncoder.java

```diff
@@ -169,7 +169,7 @@ private static int parseOffset(String group, String offsetName, String columnNam
         private static FieldType parseFieldType(String dataFormat, String columnName)
         {
             try {
-                if (!dataFormat.equals("")) {
+                if (dataFormat.length() > 0) {
```

Comment:
What about this variant?

```suggestion
                if (!dataFormat.isEmpty()) {
```